<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardIntro</class>
 <widget class="QWidget" name="WizardIntro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16pt;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Select the Import Method:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="SteamRadio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Import Your Steam Library</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="ManualRadio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Import a Single Game Manually</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="EpicRadio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;</string>
   </property>
   <property name="text">
    <string>Import Your Epic Games Library</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
